auto_nag/scripts/component.py

```diff
@@ -3,18 +3,21 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from bugbug.models.component import ComponentModel
+from libmozdata.bugzilla import Bugzilla
 
 from auto_nag import logger
-from auto_nag.bugbug_utils import BugbugScript
+from auto_nag.bugbug_utils import get_bug_ids_classification
+from auto_nag.bzcleaner import BzCleaner
 from auto_nag.utils import nice_round
 
 
-class Component(BugbugScript):
+class Component(BzCleaner):
     def __init__(self):
         self.model_class = ComponentModel
         super().__init__()
         self.autofix_component = {}
         self.frequency = "daily"
+        self.to_cache = set()
```

Comment:
I think we should be able to remove the custom cache now, @calixteman do you remember why we had the custom cache for bugbug scripts?

Reply:
I think it's likely related to this comment https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/bugbug_utils.py#L64

Reply:
Yes, that was it!
I guess now we don't care so much anymore, as the http server has its own cache.